cos/static/turns.js

```diff
@@ -8,6 +8,7 @@ var settlement_animation;
  * Called when backend confirms it is player's turn
  */
 function start_turn() {
+    check_for_winner(end_game);
```

Comment:
Should probably move this to after render_board() and get_player_info() so the players can see what happened during the turn to cause the win.